discord/ext/commands/bot.py

```diff
@@ -362,6 +363,48 @@ def after_invoke(self, coro):
 
         self._after_invoke = coro
         return coro
+    
+    def cooldown_bypass(self, coro):
+        """A decorator the registers a coroutine as a cooldown bypass check.
+
+        A cooldown bypass is a check that determines whether command cooldowns should
+        be bypassed or not. If the check returns True, cooldowns will be bypassed.
+        Elsewise if False, cooldowns will run as normal and ratelimits will apply.
+
+        This applies to all commands and their cooldowns.
+        The cooldown_bypass takes a sole parameter, a :class:`.Context`.
+
+        Parameters
+        -----------
+        coro: :ref:`coroutine <coroutine>`
+            The coroutine to register as a cooldown bypass check.
+
+        Example
+        ---------
+
+        .. code-block:: python3
+
+            @bot.cooldown_bypass
+            async def my_bot_cooldown_bypass(ctx):
+                if ctx.author == ctx.guild.owner:
+                    return True
+                return False
+
+            @bot.command()
+            @commands.cooldown(1, 30, type=commands.BucketType.guild)
+            async def my_command(ctx):
+                await ctx.send('This message can be sent unlimitedly by the guild owner, due to my cooldown bypass.)
+
+        Raises
+        -------
+        discord.ClientException
+            The coroutine is not actually a coroutine.
```

Comment:
```suggestion
            The coroutine passed is not actually a coroutine.
```

Same everywhere else.